src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -1,24 +1,96 @@
 package seedu.address.logic.commands;
-
+import seedu.address.model.person.Role;
+import seedu.address.model.person.Contact;
+import seedu.address.model.person.Tutorial;
+import seedu.address.model.person.Course;
 import static java.util.Objects.requireNonNull;
 import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
-
+import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
+import java.util.ArrayList;
+import java.util.Set;
 
 /**
  * Lists all persons in the address book to the user.
  */
 public class ListCommand extends Command {
-
+    AddressBook ab = new AddressBook();
     public static final String COMMAND_WORD = "list";
+    public static final String enter = "\n      ";
+    public static int numPeople = 0;
+    public static String s = "";
+    ArrayList<Object> nameList = new ArrayList<>();
+    ArrayList<Set<Role>> roleList = new ArrayList<>();
+    ArrayList<Set<Contact>> contactList = new ArrayList<>();
+    ArrayList<Set<Course>> courseList = new ArrayList<>();
+    ArrayList<Set<Tutorial>> tutorialList = new ArrayList<>();
+
+    String roleString(Set<Role> roleList) {
```

Comment:
it is a good practice to add access modifiers (i.e. public, private, protected, etc.) and to name methods as a verb e.g. toRoleString(...)